mulog-kinesis/test/com/brunobonacci/mulog/publishers/kinesis/test_publisher.clj

```diff
@@ -44,19 +40,18 @@
        (sp#))))
```

Comment:
To be more complete this should create a stream with a prefix like `mulog-test-` followed by a UUID for example.
wait until the stream is available, then start the publisher, run the test and consume the items and destroy the stream
This would allow to remove the face facts
https://github.com/BrunoBonacci/mulog/blob/master/mulog-kinesis/test/com/brunobonacci/mulog/publishers/kinesis/kinesis_test.clj#L9
and
https://github.com/BrunoBonacci/mulog/blob/master/mulog-kinesis/test/com/brunobonacci/mulog/publishers/kinesis/kinesis_test.clj#L53